Assets/Scripts/RandomStartAngle.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+public class RandomStartAngle : MonoBehaviour
+{
+
+    private void Start()
+    {
```

Comment:
`Random.rotation` returns a random quaternion. No need to do this.

Reply:
Removed this file since you'll take care of it in procedural generation